README.md

```diff
@@ -93,6 +118,10 @@ please take a look at https://docs.yugabyte.com/latest/reference/configuration/y
     yb_tserver_ui_port: 9000
     ysql_port: 5433
     ycql_port: 9042
+
+# Sample usage:
+- name: Test YB Tserver UI
+  run: curl --head "http://localhost:9000"
```

Comment:
It might be possible someone is looking to test when SQL and CQL have credentials, so please add those verification steps that work well with credentials. You'll find those steps in action itself.

Reply:
Done in 31686cf.